product/dialogs/miq_dialogs/miq_provision_openstack_dialogs_template.yaml

```diff
@@ -173,6 +182,15 @@
           :required_method: :validate_cloud_network
           :display: :edit
           :data_type: :integer
+        :network_port:
+          :values_from:
+            :method: :allowed_network_ports
```

Comment:
I don't see this method in the manageiq code base

Reply:
It exists in manageiq-providers-openstack (once the associated PR is pushed). This is an openstack-specific yaml file